templates/partial/contact_info.html

```diff
@@ -1,20 +1,30 @@
 <div class="govuk-body">
   <h2 class="govuk-heading-s govuk-!-margin-bottom-1">IAO or Data Owner</h2>
   <p class="govuk-body">
-    {{data_owner}}<br>
-    {{data_owner_email|urlize}}
+    {{ data_owner }}<br>
+    {{ data_owner_email|urlize }}
   </p>
 </div>
 <div class="govuk-body">
-  <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Contact for access requests</h2>
+  <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Contact email for access requests</h2>
   <p class="govuk-body">
-    {{data_owner_email|urlize}}
+    {{ data_owner_email|urlize }}
   </p>
 </div>
+<!-- placeholder until we have more contact channels than just slack -->
+{% if slack_channel.dc_slack_channel_url %}
+  <div class="govuk-body">
+    <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Contact channels for access requests</h2>
+    <p class="govuk-body">
+        <!-- This should become a list of populated contact channels -->
+      Slack channel: <a href="{{ slack_channel.dc_slack_channel_url }}" target="_blank">{{ slack_channel.dc_slack_channel_name }}</a>
```

Comment:
[The design system generally recommends against links that open in a new tab](https://design-system.service.gov.uk/styles/links/#:~:text=link%20icon.-,Opening%20links%20in%20a%20new%20tab,-Avoid%20opening%20links)

If we keep the `target="_blank"`, we should also include "(opens in new tab)" in the link text.

Reply:
Happy to add the extra text. Depends on expected behaviour - I would always expect a slack / teams link to open in a new window as it likely leads to migrating to a site outside of the browser.